RiotSharp/Endpoints/StaticDataEndpoint/TarballLinks/StaticTarballLinkEndPoint.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using RiotSharp.Caching;
+using RiotSharp.Endpoints.Interfaces.Static;
+using RiotSharp.Http.Interfaces;
+using RiotSharp.Misc;
+
+namespace RiotSharp.Endpoints.StaticDataEndpoint.TarballLinks
+{
+    public class StaticTarballLinkEndPoint : StaticEndpointBase, IStaticTarballLinkEndPoint
+    {
+        private const string TarballLinksUrl = "tarball-links";
+        private const string TarballLinksCacheKey = "tarball-links";
+
+        public StaticTarballLinkEndPoint(IRateLimitedRequester requester, ICache cache)
+            : base(requester, cache) { }
+
+        public StaticTarballLinkEndPoint(IRateLimitedRequester requester, ICache cache, TimeSpan? slidingExpirationTime)
+            : base(requester, cache, slidingExpirationTime) { }
+
+        public async Task<string> GetTarballLinksAsync(Region region)
```

Comment:
Could you remove this method and instead make the `version` parameter in `GetTarballLinksAsync` optional?

Reply:
Np, I'm looking into it.